.github/workflows/release-preview.yaml

```diff
@@ -26,6 +26,9 @@ jobs:
 
       - name: Patch to preview version
         run: npm run patch-preview
+        env:
+          SNYK_VSCE_SEGMENT_WRITE_KEY: ${{ secrets.SNYK_VSCE_SEGMENT_WRITE_KEY }}
+          SNYK_VSCE_AMPLITUDE_EXPERIMENT_API_KEY: ${{ secrets.SNYK_VSCE_AMPLITUDE_EXPERIMENT_API_KEY }}
```

Comment:
@michelkaporin, do we use the same keys for preview build? Will you be able to distinguish the versions?

Reply:
@pavel-snyk we use different keys. GitHub environment has the correct keys specified for Preview
<img width="775" alt="Screenshot 2022-01-06 at 11 49 53" src="https://user-images.githubusercontent.com/2239563/148371539-5f903201-b6c2-4198-ba17-cf01269e4d2f.png">. It doesn't store any for stable (since no CD for it available)

Reply:
got it, thanks :)